src/assets/images/info.svg

```diff
@@ -0,0 +1 @@
+<svg height='100px' width='100px'  fill="#000000" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 450 450" x="0px" y="0px"><title>Fichier 2locate</title><g data-name="Calque 2"><g data-name="Calque 2"><circle cx="225" cy="133.5" r="31.25"></circle><path d="M251,219.12v103a26,26,0,0,1-52,0v-103a26,26,0,0,1,52,0Z"></path><path d="M225,0C100.74,0,0,100.74,0,225S100.74,450,225,450,450,349.26,450,225,349.26,0,225,0ZM355.81,355.81A185,185,0,1,1,94.19,94.19,185,185,0,1,1,355.81,355.81Z"></path><path d="M225,0C100.74,0,0,100.74,0,225S100.74,450,225,450,450,349.26,450,225,349.26,0,225,0ZM355.81,355.81A185,185,0,1,1,94.19,94.19,185,185,0,1,1,355.81,355.81Z"></path></g></g></svg>
```

Comment:
There's an already existing svg called `exclamation-circle-solid` which was being used in that message "only indian numbers allowed"
either use that here or use this one there, no need for two different svgs